staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -47,7 +47,11 @@ type WardleServerOptions struct {
 
 func NewWardleServerOptions(out, errOut io.Writer) *WardleServerOptions {
 	o := &WardleServerOptions{
-		RecommendedOptions: genericoptions.NewRecommendedOptions(defaultEtcdPathPrefix, apiserver.Codecs.LegacyCodec(v1alpha1.SchemeGroupVersion)),
+		RecommendedOptions: genericoptions.NewRecommendedOptions(
+			defaultEtcdPathPrefix,
+			apiserver.Codecs.LegacyCodec(v1alpha1.SchemeGroupVersion),
+			genericoptions.NewProcessInfo("wardle", "kube-system"),
```

Comment:
if someone's going to copy/paste the sample apiserver and forget to change the namespace, I guess I'd rather they try to hit wardle, fail, and investigate than assume write permissions to kube-system. "wardle-apiserver" in the "wardle" namespace works for now. 

kube-system is good for all the apiservers built into kube-apiserver.